desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -690,6 +690,12 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
                 </label>
               </div>
 
+              <div class="control-group" data-bind="visible: !useDefaultLocation() && !isTransactional() && $root.createWizard.source.inputFormat() === 'file'">
+                <label class="checkbox inline-block">
+                  <input data-hue-analytics="importer:useCopy-checkbox-interaction" type="checkbox" data-bind="checked: useCopy"> ${_('Copy file')}
```

Comment:
How about we add an info icon with a details popover that explains to the user what this means?

Reply:
Sure, looks good to add the details.
I added a popover accordingly, not sure is it correct way to add, please correct it if it is wrong.
![Screenshot 2023-11-02 at 9 21 56 AM](https://github.com/cloudera/hue/assets/36241930/03333114-e6f0-4655-b23e-21fe38081a26)